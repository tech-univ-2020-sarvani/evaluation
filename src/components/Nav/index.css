.NavBar {
    background-color: rgb(93, 201, 59);
    display: flex;
    height: 4rem;
    flex-direction: row;
    justify-content: flex-start;
    align-items: center;
    font-family: 'Poppins';
    color: white;
    font-size:xx-large;
}