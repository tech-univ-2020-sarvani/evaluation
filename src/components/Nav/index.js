/* eslint-disable react/require-default-props */
import React from 'react';
import './index.css';
import PropTypes from 'prop-types';

function Nav(props) {
  const { testId } = props;
  return (
    <div className="NavBar" data-testid={testId}>
      <p>
        <b>Record</b>
        Shelf
      </p>
    </div>
  );
}
Nav.propTypes = {
  testId: PropTypes.string,
};

export default Nav;
