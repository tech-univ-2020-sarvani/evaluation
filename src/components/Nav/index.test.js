import React from 'react';
import { render } from '@testing-library/react';
import Nav from './index';

describe('the Nav component', () => {
  it('should render correctly', () => {
    const { asFragment } = render(<Nav testId="test-btn" />);

    expect(asFragment()).toMatchSnapshot();
  });
});
