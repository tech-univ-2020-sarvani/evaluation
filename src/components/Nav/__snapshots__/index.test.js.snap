// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the Nav component should render correctly 1`] = `
<DocumentFragment>
  <div
    class="NavBar"
    data-testid="test-btn"
  >
    <p>
      <b>
        Record
      </b>
      Shelf
    </p>
  </div>
</DocumentFragment>
`;
